README.md

```diff
@@ -124,7 +124,7 @@ See, the `publish` tool expects a number of environment variables to be set.
 These are:
 
 ```console
-DATAPUNT_ENVIRONMENT=[acceptation|production]
+DATAPUNT_ENVIRONMENT=[acceptation|production|...]  # default is 'acceptance'
```

Comment:
"acceptation" of "acceptance"?

Reply:
Good catch. Het moet `acceptance` zijn, pas ik aan.